ActivityPub.Types/ASType.cs

```diff
@@ -31,17 +31,17 @@ public abstract class ASType
     /// Should be a full URL
     /// </summary>
     [JsonPropertyName("@context")]
-    [JsonConverter(typeof(JsonLDContextPropertyConverter))]
-    public HashSet<JsonLDContext> JsonLdContexts { get; set; } = new()
+    [JsonConverter(typeof(JsonLDContextConverter))]
+    public JsonLDContext JsonLdContexts { get; set; } = new(new HashSet<JsonLDContextObject>
     {
         // We always need the base context
-        ActivityStreamsContext
-    };
-
+        ActivityStreamsContextObject
+    });
+    
     /// <summary>
     /// Shared JSON-LD context used by all ActivityStreams objects.
     /// </summary>
-    public static JsonLDContext ActivityStreamsContext { get; } = new("https://www.w3.org/ns/activitystreams");
+    public static JsonLDContextObject ActivityStreamsContextObject { get; } = new("https://www.w3.org/ns/activitystreams");
```

Comment:
Can we rename this to just `ActivityStreamsContext`? In the future, I'm planning to upgrade `JsonLDContext` to manage the underlying objects and terms directly. At that point, this property will change to a `string` or `Uri` (not sure which just yet) and the "Object" suffix will be inappropriate.